README.md

```diff
@@ -2662,6 +2662,10 @@ command = "echo"
 args = [ "running in production profile" ]
 ```
 
+#### Default profile
```

Comment:
can you put this in the content.md instead of readme as readme is generated?
also please add an anchor like link above and a link to this anchor in nav.md